### PR TITLE
Adjust fieldset spacing

### LINKED ISSUE #455: Form element spacing
Screen Shot 2020-10-01 at 9 55 53 AM

### PR DESCRIPTION
**Pull request by mmazanec22**
Closes #455 
Depends on https://github.com/ctoec/component-library/pull/77

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@ctoec/component-library": "https://github.com/ctoec/component-library#base",
+    "@ctoec/component-library": "https://github.com/ctoec/component-library#fieldset-spacing",
```
Comment (bamader) [inline client/package.json:6]: I think this should go back to base before merging

File: client/yarn.lock [hunk 2]
```diff
@@ -1166,9 +1166,9 @@
   resolved "https://registry.yarnpkg.com/@csstools/normalize.css/-/normalize.css-10.1.0.tgz#f0950bba18819512d42f7197e56c518aa491cf18"
   integrity sha512-ij4wRiunFfaJxjB0BdrYHIH8FxBJpOwNPhhAcunlmPdXudL1WQV1qoP9un6JsEBAgQH+7UXyyjh0g7jTxXK6tg==
 
-"@ctoec/component-library@https://github.com/ctoec/component-library#base":
+"@ctoec/component-library@https://github.com/ctoec/component-library#fieldset-spacing":
```
Comment (bamader) [inline client/yarn.lock:1169]: Back to base before merging. Gonna go review the PRs up in the component library so we can get those merged, too.

**Review by bamader — APPROVED** [review]
LGTM! We just need to switch the component library branch back to base before merging.

File: client/package.json (hunk 1 above)
Comment (jlhogan) [reply client/package.json:6]: yes, i think at this point we can just assume that's the plan :)

**Review by jlhogan — APPROVED** [review]

**Pull request merged**